image.yaml

```diff
@@ -0,0 +1,133 @@
+schema_version: 1
+
+name: "jboss-eap-7-tech-preview/eap74-openjdk11-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform 7.4.0.Beta OpenShift container image."
+version: "7.4.0.Beta"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-beta-openjdk11-builder-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for building and running JavaEE applications on JBoss EAP 7.4.0.Beta"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP 7.4.0.Beta"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "builder,javaee,eap,eap7"
+    - name: "io.openshift.s2i.scripts-url"
+      value: "image:///usr/local/s2i"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+    - name: HTTPS_ENABLE_HTTP2
+      value: "true"
+    - name: SCRIPT_DEBUG
+      description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+      example: "true"
+ports:
+    - value: 8443
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: 0.44.0
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/luck3y/jboss-eap-modules.git
+                  ref: CLOUD-3918
+          - name: jboss-eap-7-image
+            git:
+                  url: https://github.com/luck3y/jboss-eap-7-image.git
+                  ref: CLOUD-3918
+          - name: wildfly-cekit-modules
+            git:
+                  url: https://github.com/luck3y/wildfly-cekit-modules.git
+                  ref: CLOUD-3918
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "11"
+          - name: jboss.container.maven
+            version: "8.2.3.6"
+          - name: eap-74-beta-env-latest
```

Comment:
The correct env is: eap-74-beta-galleon-latest

Reply:
Thanks, actually this is what I had locally, i just hadn't pushed it, corrected.